config/application.yml.example

```diff
@@ -1,3 +1,16 @@
+DATABASE_URL: 'postgres://localhost:5432/rails_new_development'
 HOST: 'http://localhost:3000'
-RAILS_WEB_CONCURRENCY: '1'
 RAILS_MAX_THREADS: '1'
+RAILS_WEB_CONCURRENCY: '1'
+REDIS_URL: 'redis://localhost:6379'
+
+# When using Docker use this instead:
```

Comment:
💯   on having a comment about the very frequently asked question about using Docker with Rails and friends